src/CRC32.jl

```diff
@@ -66,6 +75,21 @@ function _crc32(s::Union{String, SubString{String}}, crc::UInt32=0x00000000)
     unsafe_crc32(s, sizeof(s) % Csize_t, crc)
 end
 
+function _crc32(a::AbstractVector{UInt8}, crc::UInt32=0x00000000)
+    # use block size 24576=8192*3, since that is the threshold for
+    # 3-way parallel SIMD code in the underlying jl_crc32 C function.
+    last::Int64 = lastindex(a)
+    nb::Int64 = length(a)
```

Comment:
```suggestion
    last::Int = lastindex(a)
    nb::Int = length(a)
```
Array lengths can't be longer than an `Int` (unlike file lengths).

Reply:
Looking at this again, I think I can completely remove the type annotations.